sim/paladin/paladin.go

```diff
@@ -105,12 +136,12 @@ func (paladin *Paladin) Initialize() {
 	paladin.registerAuraMastery()
 
 	paladin.lingerDuration = time.Millisecond * 400
+	paladin.consumeSealsOnJudge = true
 
 	paladin.registerStopAttackMacros()
 }
 
 func (paladin *Paladin) Reset(_ *core.Simulation) {
-	//paladin.registerStopAttackMacros()
 }
```

Comment:
WIth `lastJudgement` we'll want to make sure to set it back to `nil` on reset here so that the value doesn't persist across iterations. That would treat the first judgement cast using the `lastJudgement` logic

Reply:
sure